example/docker-compose.yml

```diff
@@ -0,0 +1,74 @@
+version: "3"
+
+services:
+  mongodb:
+    image: mongo:3.2
+    # We use WiredTiger in all environments. In development environments we use small files
+    # to conserve disk space, and disable the journal for a minor performance gain.
+    # See https://docs.mongodb.com/v3.0/reference/program/mongod/#options for complete details.
+    command: mongod --smallfiles --nojournal --storageEngine wiredTiger
+
+  mailcatcher:
+    image: sj26/mailcatcher:latest
+    ports:
+      - "1080:1080"
+
+  redis:
+    image: redis:4-alpine
+
+  xapi:
+    image: learninglocker/xapi-service:2.2.14
+    restart: unless-stopped
+    environment:
+      - MONGO_URL=mongodb://mongodb:27017/learninglocker_v2
+      - REDIS_URL=redis://redis:6379/0
+    depends_on:
+      - mongodb
+      - redis
+      - api
+      - ui
+      - worker
+
+  api:
+    image: fundocker/learninglocker:v2.6.2
```

Comment:
Why not using the `local` tag to use local builds?

Reply:
Because I want to use a published image